lib/ui/huge_listview/lazy_loading_gallery.dart

```diff
@@ -237,19 +294,29 @@ class _LazyLoadingGridViewState extends State<LazyLoadingGridView> {
 
   @override
   void initState() {
-    super.initState();
     _shouldRender = widget.shouldRender;
-    widget.selectedFiles.addListener(() {
-      bool shouldRefresh = false;
-      for (final file in widget.files) {
-        if (widget.selectedFiles.isPartOfLastSection(file)) {
-          shouldRefresh = true;
+    widget.shouldSelectAll.addListener(() {
```

Comment:
Do we need to remove this listener? If not, can you please add a minor comment with the reason?

Reply:
Accidentally left this out, have removed it.